src/main/java/com/google/firebase/FirebaseApp.java

```diff
@@ -163,6 +170,42 @@ public static FirebaseApp getInstance(@NonNull String name) {
     }
   }
 
+  /**
+   * Initializes the default {@link FirebaseApp} instance. Same as {@link
+   * #initializeApp(String)}, but it uses {@link #DEFAULT_APP_NAME} as name. *
+   * Also, it uses the {@link FirebaseOptions} values set in the file pointed to by
+   * {@link DEFAULT_CONFIG_OPTIONS_VAR}, if tht file is available.
+   * 
+   * <p>The creation of the default instance is automatically triggered at app startup time, if
```

Comment:
Oh! I think this is some left over from Android SDK. Please remove this paragraph -- here and elsewhere in this class.